packages/components/modal/src/modalTypes/ModalHeader.tsx

```diff
@@ -0,0 +1,47 @@
+import { ModalHeaderProps, ModalStyledProps } from '../modal.types';
+import { ModalHeaderStyled, ModalIconStyled } from '../modal.styles';
+import { deepMerge, useTheme } from '@douro-ui/react';
+import { Close } from '@douro-ui/svg-icons';
```

Comment:
When using anything form outside of the package, needs to be added to the package.json dependencies